circuits/comparators.circom

```diff
@@ -98,6 +98,23 @@ template LessThan(n) {
     out <== 1-n2b.out[n];
 }
 
+// Do Range checks on the inputs to avoid overflow
+template SafeLessThan(n) {
+    assert(n <= 252);
+    signal input in[2];
+    signal output out;
+
+    component aInRange = Num2Bits(252);
```

Comment:
We can save some constraints by restricting to `n` bits instead. It will always be safe since `n <= 252` anyway. Also, that way we can interpret `SafeLessThan(n)` as meaning "safely compare two n-bit numbers." This makes it easier to understand why `in[0]+ (1<<n) - in[1]` should be an n+1 bit number.

Reply:
Thanks for the comment. Good point. Added to the PR.